R/aadoc.R

```diff
@@ -1,2 +1,19 @@
 #### Documentation objects
 
+#' Identifiers for MEMC supported kinetics used
+#'
+#' These identifiers correspond to the various kinetics users can use in \code{\link{carbon_fluxes}} to
+#' select for different kinetic dynamics.
+#'
+#' @section Kinetics:
+#' \describe{
+#' \item{MM}{Michaelis–Menten kinetics, TODO add ref}
+#' \item{RMM}{Reverse Michaelis-Menten kinetics, TODO add ref}
+#' \item{ECA}{Equilibrium chemistry approximation, based on RESOM (Tang and Riley, 2014)}
+#' \item{LM}{Linear kinetics model, TODO add ref}
+#' }
```

Comment:
@jianqiuz do we need to add references for the Michaelis–Menten kinetics, Reverse Michaelis-Menten kinetics ad, linear kinetics model?

Reply:
RMM ref: Schimel, J. P., and M. N. Weintraub (2003), The implications of exoenzyme activity on microbial carbon and nitrogen limitation in soil: A theoretical model, Soil Biol. Biochem., 35(4), 549–563, doi:10.1016/s0038-0717(03)00015-4; Sulman, B. N., R. P. Phillips, A. C. Oishi, E. Shevliakova, and S. W. Pacala (2014), Microbe-driven turnover offsets mineral-mediated storage of soil carbon under elevated CO2, Nat. Clim. Change, 4(12), 1099–1102, doi:10.1038/nclimate2436.

Linear ref: Robertson, A. D., Paustian, K., Ogle, S., Wallenstein, M. D., Lugato, E., and Cotrufo, M. F.: Unifying soil organic matter formation and persistence frameworks: the MEMS model, Biogeosciences, 16, 1225–1248, https://doi.org/10.5194/bg-16-1225-2019, 2019.

Reply:
Awesome thanks!